src/components/sections/Faqs.css

```diff
@@ -0,0 +1,13 @@
+.collapse {
+  @apply rounded-lg border border-gray-300 bg-white;
+}
+
+.collapse-title {
+  @apply cursor-pointer text-xl font-medium;
+}
+
+.collapse,
+.collapse-title,
+.collapse-content * {
+  @apply text-black;
+}
```

Comment:
Don't need a new css file for @apply, can just apply these styles inline using tailwind.

e.g
```tsx
 <p className="text-black">{faq.answer}</p>
```